### PR TITLE
Start read

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,95 @@
...
+In this Code Pattern, we will use fictional sales data to create a logistic regression model using Azure Machine Learning Studio. We will use AI OpenScale to bind the ML model deployed in the Azure cloud, create a subscription, and perform payload and feedback logging.
+
+When the reader has completed this Code Pattern, they will understand how to:
+
+* Prepare data, train a model, and deploy using Azure Machine Learning Studio
+* Score the model using sample scoring records and the scoring endpoint
+* Setup AI OpenScale Data Mart
+* Bind the Azure model to the AIOS Data Mart
+* Add subscriptions to the Data Mart
+* Enable payload logging and performance monitor for both subscribed assets
```
Comment (rhagarty) [inline README.md]: use "monitoring" here?

File: README.md [hunk 2]
```diff
@@ -1 +1,95 @@
...
+* Using the [IBM Cloud Dashboard](https://console.bluemix.net/catalog) catalog, search for PostgreSQL and choose the `Compose for Postgres` service:
+
+![chooseComposePostgres](doc/source/images/ChooseComposePostgres.png)
+
+* Wait a couple of minutes for the database to be provisioned.
+* Click on the `Service Credentials` tab on the left and then click `New credential +` to create the service credentials. Copy them or leave the tab open to use later in the notebook.
+
+### 3. Create an AI OpenScale service
+
+* Using the [IBM Cloud Dashboard]() create an [AI OpenScale](https://console.bluemix.net/catalog/services/ai-openscale) service.
```
Comment (rhagarty) [inline README.md]: add dashboard link

**Review by rhagarty — CHANGES_REQUESTED** [review]
nits

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: done

**Review by rhagarty — APPROVED** [review]

**Pull request merged**